### PR TITLE
data: lux5h

### PR DESCRIPTION
**Pull request by lux5h**

### CODE REVIEW

File: data/lux5h.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    {
+      "name": "Follow on Twitter",
+      "url": "https://twitter.com/Ar_Shreelaxmi",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/ar-shreelaxmi/",
+      "icon": "FaLinkedIn"
+    },
```
Comment (senali-d) [inline data/lux5h.json:17]: Please remove the `,`
```suggestion
    }
```

**Review by senali-d — CHANGES_REQUESTED** [review]

**Comment by ChinmayMhatre** [comment]
@lux5h so basically once you create a PR using a branch and someone suggests changes, you can commit the changes to the same branch and they will reflect here in the PR. I noticed you created a new PR with the changes, try committing them to the same branch next time, Thanks!

**Comment by lux5h** [comment]
> PR

Thank you! will keep in mind from next time.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**